src/ivoc/ivocvect.cpp

```diff
@@ -1192,6 +1192,9 @@ static Object** v_smhist(void* v) {
 
         if (ifarg(6)) {
 	   w = vector_arg(6);
+        if (w->size() != data->size()) {
```

Comment:
Found this because we are now using `std::vector::at()`, which does bounds checking. so before with `gnuvec` we were reading memory from somewhere else if the size wasn't the same ..